### DISCUSSION TITLE
Managed hosting option

### DISCUSSION THREAD

**Discussion by m3nu** [lukevella/rallly#660] [Ideas]
Hey,

Manu here from [PikaPods](https://www.pikapods.com/). I think we already talked last year, when we first added Rallly. Some of our users pointed out that we've been neglecting updates a bit with the fast pace of development since then. So I'm happy to have all our user's pods on the latests version and all new env vars are now also available. We'll keep rolling out updates, as they are published here.

I also wanted to suggest adding PikaPods as managed hosting option in the docs or README somewhere. This would help an even wider audience to benefit from the project and quickly run their own Rallly pod. Could be a button like this:

[![Run on PikaPods](https://www.pikapods.com/static/run-button.svg)](https://www.pikapods.com/pods?run=rallly)

We also do revenue shares with projects we host, which we can offer as well. In addition to featuring you in our monthly newsletter or homepage.

Looking forward to your opinion on this!

**Comment by m3nu** [comment]
One more thing: the `SUPPORT_EMAIL` env var is missing from the README: https://github.com/lukevella/rallly#configuration-options

Noticed after accidentally removing it and then adding it back. Worth opening an issue for this?

**Reply by lukevella** [reply]
Hey @m3nu, thanks for pointing that out, I'll get on to fixing that straight away, no need to open an issue.

To make an informed decision about adding your badge to the README, I would require specific figures. Please understand that I am unable to extend favors at this time, and if your proposed compensation is only a small amount each month, it may not be viable for me.

**Reply by m3nu** [reply]
The amount depends on the number of users really. Some of our authors reach $500 a quarter. Of course if nobody knows about the option they won't use it. A bit of a chicken-egg problem really.

So until there are more users, I can only throw in the newsletter option, which goes to about 4000 of our users every month. We often feature new or updated apps there. Maybe Jeremy over at https://noted.lol/ wants to do an article as well. We often work together with him.

Sorry I can't offer a proper sponsorship for now. The project is still growing and we keep prices affordable, so only the revenue share is viable for now.

**Reply by lukevella** [reply]
I can probably work this in but I have a couple updates I need to get out first. Is it ok if I reach out to you in a week or two?

**Reply by m3nu** [reply]
Sure thing. I'm here and will keep an eye out for any response here.

**Reply by lukevella** [reply]
Hey @m3nu, so I have this page ready in our docs for listing managed hosting providers:

https://support.rallly.co/self-hosting/managed-hosting

Happy for you to be first on the list. I'll eventually have other providers on this list as well so I'd like to include information such as the revenue share you are offering to encourage users to use services like yours that contribute something back to the project. Can you let me know what revenue share you are offering so I can include that info?

Thanks.

**Reply by m3nu** [reply]
Looks good! Already saw it last week.

Standard revenue share is 20% paid quarterly with monthly reporting on how many pods and deployed revenue.

**Reply by lukevella** [reply]
Hey @m3nu, it's up now: [link](https://support.rallly.co/self-hosting/managed-hosting)

Can you let me know if there's anything I need to do to get the monthly reporting and the revenue share.

Thanks!

**Reply by m3nu** [reply]
Great! Please email your PayPal email and email to use for the monthly report to `hello at pikapods.com`. Then I'll add it to the database. Thanks!
